### PR TITLE
docs: Consul K8s 0.49.x release notes

### PR DESCRIPTION
**Pull request by nrichu-hcp**
### Description
0.49.x Release Notes

### Testing & Reproduction steps
* In the case of bugs, describe how to replicate
* If any manual tests were done, document the steps and the conditions to replicate
* Call out any important/ relevant unit tests, e2e tests or integration tests you have added or are adding

### Links
Include any links here that might be helpful for people reviewing your PR (Tickets, GH issues, API docs, external benchmarks, tools docs, etc). If there are none, feel free to delete this section.

Please be mindful not to leak any customer or confidential information. HashiCorp employees may want to use our internal URL shortener to obfuscate links.

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] not a security concern

### CODE REVIEW

File: website/content/docs/release-notes/consul-k8s/v0_49_x.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+# Consul on Kubernetes 0.49.0
+
+## Release Highlights
+
+- **Consul CNI Plugin - OpenShift support**: Support for OpenShift and Multus CNI plugin [GH-1527]
+
+- **Consul API Gateway secondary datacenter support**: Use global ACL auth method to provision ACL tokens for API Gateway in secondary datacenter and Set primary datacenter flag when deploying controller into secondary datacenter with federation enabled [GH-1481]
+
+- **Cluster Peering**: pass new use_auto_cert value to gRPC TLS config when auto-encrypt is enabled. [GH-1541]
```
Comment (david-yu) [inline website/content/docs/release-notes/consul-k8s/v0_49_x.mdx:16]:
```suggestion
- **Cluster Peering**: Utilize new `use_auto_cert` value to gRPC TLS config when auto-encrypt is enabled. [GH-1541]
```

File: website/content/docs/release-notes/consul-k8s/v0_49_x.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  [Envoy Integration](/docs/connect/proxies/envoy) for details.
+
+## Upgrading
+
+For detailed information on upgrading, please refer to the [Upgrades page](/docs/k8s/upgrade)
+
+## Known Issues
+The following issues are know to exist in the v0.49.0 release: 
+
+- Kubernetes 1.25 is not supported as the [pod security admission controller](https://kubernetes.io/blog/2022/08/25/pod-security-admission-stable/) is not supported by Consul K8s.
```
Comment (david-yu) [inline website/content/docs/release-notes/consul-k8s/v0_49_x.mdx:38]:
```suggestion
- Kubernetes 1.25 is not supported as the [Pod Security Admission controller](https://kubernetes.io/blog/2022/08/25/pod-security-admission-stable/) is currently not supported by Consul K8s. 
```

**Review by david-yu — COMMENTED** [review]
Looks good. Added some suggestions. Also include the Github PR links if possible.

**Comment by nrichu-hcp** [comment]
Added recommended changes

File: website/content/docs/release-notes/consul-k8s/v0_49_x.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+page_title: 0.49.x
+description: >-
+  Consul on Kubernetes release notes for version 0.49.x
+---
+
+# Consul on Kubernetes 0.49.0
+
+## Release Highlights
+
+- **Consul CNI Plugin - OpenShift support**: Support for OpenShift and Multus CNI plugin [GH-1527](https://github.com/hashicorp/consul-k8s/pull/1527)
```
Comment (david-yu) [inline website/content/docs/release-notes/consul-k8s/v0_49_x.mdx:12]:
```suggestion
- **Consul CNI Plugin - OpenShift support**: Support for OpenShift and Multus CNI plugin [[GH-1527](https://github.com/hashicorp/consul-k8s/pull/1527)]
```

File: website/content/docs/release-notes/consul-k8s/v0_49_x.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+- **Consul API Gateway secondary datacenter support**: Use global ACL auth method to provision ACL tokens for API Gateway in secondary datacenter and Set primary datacenter flag when deploying controller into secondary datacenter with federation enabled [GH-1481](https://github.com/hashicorp/consul-k8s/pull/1481)
```
Comment (david-yu) [inline website/content/docs/release-notes/consul-k8s/v0_49_x.mdx:14]:
```suggestion
- **Consul API Gateway secondary datacenter support**: Use global ACL auth method to provision ACL tokens for API Gateway in secondary datacenter and Set primary datacenter flag when deploying controller into secondary datacenter with federation enabled [[GH-1481](https://github.com/hashicorp/consul-k8s/pull/1481)]
```

File: website/content/docs/release-notes/consul-k8s/v0_49_x.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+- **Cluster Peering**: Utilize new `use_auto_cert` value to gRPC TLS config when auto-encrypt is enabled. [GH-1541](https://github.com/hashicorp/consul-k8s/pull/1541)
```
Comment (david-yu) [inline website/content/docs/release-notes/consul-k8s/v0_49_x.mdx:16]:
```suggestion
- **Cluster Peering**: Utilize new `use_auto_cert` value to gRPC TLS config when auto-encrypt is enabled. [[GH-1541](https://github.com/hashicorp/consul-k8s/pull/1541)]
```

File: website/content/docs/release-notes/consul-k8s/v0_49_x.mdx [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+- **Service tag annotation improvements**: Support escaped commas in service tag annotations for pods which use consul.hashicorp.com/connect-service-tags or consul.hashicorp.com/service-tags. [GH-1532](https://github.com/hashicorp/consul-k8s/pull/1532)
```
Comment (david-yu) [inline website/content/docs/release-notes/consul-k8s/v0_49_x.mdx:18]:
```suggestion
- **Service tag annotation improvements**: Support escaped commas in service tag annotations for pods which use consul.hashicorp.com/connect-service-tags or consul.hashicorp.com/service-tags. [[GH-1532](https://github.com/hashicorp/consul-k8s/pull/1532)]
```

**Review by david-yu — APPROVED** [review]
Small formatting suggestions to add brackets back in but looks good!

**Pull request merged**